docs/user/science.rst

```diff
@@ -0,0 +1,142 @@
+
+    Design your science project's layout and components with widgets from a rich eco-system of Open Source extensions built for many purposes. Special widgets help users display and interact with graphs, maps and more. :external+jupyterbook:doc:`Several <content/components>` `extensions <https://sphinx-gallery.github.io/>`__ are built and invented by the science community.
+
+.. dropdown:: ⚙️ Automatic builds
+
+    Build and publish your project for every change made through Git (GitHub, GitLab, BitBucket etc). Preview changes via Pull Requests. Receive notifications when something is wrong. How does this work? Have a look at this video:
+
+    .. TODO: Where should this video live?
+
+    .. video:: https://overtag.dk/files/enable-pull-request-builders.mp4
+       :width: 500
+       :height: 300
+
+.. dropdown:: 💬 Collaboration and community
+
+    .. figure:: /img/screenshot_edit_on_github.png
+        :align: right
+
+    Science and academia have a big kinship with software developers: We ❤️ community. Our solutions and projects become better when we foster inclusivity and active participation. Read the Docs features easy access for readers to suggest changes via GitHub, GitLab, BitBucket etc. But not just any unqualified feedback. Instead, the *code* and all the tools are available for your community to forge qualified contributions.
+
+    Your readers can become your co-authors!
+
+    Discuss changes via Pull Request and track all changes in your project's version history.
+
+    Does this sound chaotic? Don't worry, the full ownership and permission handling remains in your hands: Project and organization owners govern what is released and who has access to approve and build changes.
+
+.. dropdown:: 🔎 Search Engine Optimization
+
+    Don't reinvent SEO: Have SEO handled jointly by the combined best-practices from Sphinx, its themes and Read the Docs' platform.
+
+.. dropdown:: 🌱 Grow your own solutions
+
+    The eco-system is Open Source and makes it accessible for anyone with Python skills to build their own extensions.
+
+We want science communities to use Read the Docs and to be part of the documentation community 💞
+
+Getting started: Jupyter Book
+-----------------------------
+
+:external+jupyterbook:doc:`Jupyter Book <intro>` on Read the Docs brings you the rich experience of computated `Jupyter <https://jupyter.org/>`__ documents built together with a modern documentation tool. The results are beautiful and automatically deployed websites, built with Sphinx and :doc:`Executable Book <executablebook:index>` + all the extensions available in this ecosystem.
+
+Here are some popular activities that are well-supported by Jupyter Book:
+
+* Publications and books
+* Course and research websites
+* Interactive classroom activities
+* Data science software documentation
+
+
+.. button-ref:: executablebook:gallery
+    :ref-type: doc
+    :color: primary
+
+    Visit the gallery of solutions built with Jupyter Book
```

Comment:
I'd not use a button for this since it just a link to an external page. I think it's more accurate to use just a sentence with a link for this.

Reply:
It's also wise to be careful with buttons. This doesn't seem like the place we want to send most readers, but a button will be really prominent.

If you want to keep this as a button, it is a very wordy button. It might be better to break up the button into a paragraph with details and a shorter button.

Reply:
It's turned into a normal link now. Not sure I'm liking how it's dangling on its own now, so I think it needs to be fixed with some other means.

I'm not too worried about people exiting to the Jupyter Book Gallery. If someone in science/academia sees this page, I feel quite confident that they will come back in order to get started. They will have already scrolled passed our feature list to reach this point.